api/src/main/java/com/capitalone/dashboard/settings/ApiSettings.java

```diff
@@ -18,7 +18,12 @@
     private boolean logRequest;
     @Value("${pageSize:10}")
     private int pageSize;
-    
+    private String gitHubWebHook;
+
+    public String getGitHubWebHook() { return gitHubWebHook; }
```

Comment:
Why can't GitHubWebHook be its own property file? its practically difficult to maintain json in a properties file. SpringBoot application can be loaded with any number of properties files. Let me know if you need help with this

Reply:
I can put this in a separate properties file, lets pass this through @tabladrum once please, thanks.